content/en/docs/Reference/configuration.md

```diff
@@ -22,6 +22,14 @@ users:
   ssh_authorized_keys:
   # - github:mudler
 
+# enable debug logging
+debug: true
```

Comment:
After the merging of all configs, does the schema need an update? https://github.com/kairos-io/kairos-sdk/tree/main/schema

Reply:
Good point! I remember updating it already so it would pass the tests. Not sure if the tests are now broken or it's not necessary anymore...

Gonna check